examples/slu/speech_intent_slot/configs/conformer_transformer_large_bpe.yaml

```diff
@@ -0,0 +1,210 @@
+# Example config for speech intent classification and slot filling with Conformer-Transformer architecture.
```

Comment:
In another PR, Add some info about config = what config combo corresponds to what model size (approx) similar to the Conformer configs.